bridge/src/bridge/gas_price.rs

```diff
@@ -44,6 +45,8 @@ impl GasPriceStream {
             speed: node.gas_price_speed,
             request_timer: timer.clone(),
             interval: timer.interval_at(Instant::now(), CACHE_TIMEOUT_DURATION),
+            last_price: node.default_gas_price,
+            request_timeout: node.request_timeout,
```

Comment:
@yrashk my expectation is that `gas_price_timeout` should be used. Since it was introduced for this. I can read the following in `README.md`:
```
home/foreign.gas_price_timeout - the number of seconds to wait for an HTTP response from the gas price oracle before using the default gas price. Defaults to 10 seconds.
```

Reply:
Yup, you are correct. That config option was added for this reason. I'm not sure how necessary having this config option it is though. Its plausible that we use a single timeout duration for all http requests (web3 and gas-price).